src/index.ts

```diff
@@ -30,7 +30,13 @@ function retryMyFetch(http: Fetch, params: decoratorOptions): Fetch {
           );
       counter += 1;
 
-      if (data.ok === true || counter > maxTryCount) return data;
+      if (
+        (watchStatus && watchStatus !== data.status) ||
```

Comment:
What if we will not return `data`, but just will not call `abortRequests()`? 
In this case, we still will be able to make a decision into `beforeRefetch` callback